ygen/gogen.go

```diff
@@ -1517,19 +1570,17 @@ func writeGoStruct(targetStruct *Directory, goStructElements map[string]*Directo
 				enumTypeMap[schemapath] = append(enumTypeMap[schemapath], mtype.NativeType)
 			}
 
-			if goOpts.GenerateLeafGetters {
```

Comment:
Do we always generate the getters in the output code, I think the answer here is yes - since these two features are now dependent upon each other? Should we just replace them with a single flag (i.e., should `generateLeafGetters` be deprecated?)

Reply:
We don't always generate the getters -- this particular guard is simply a run-time optimization, because this same guard is present when `generateLeafGetters`, i.e. the function that populates the template, is called.

I removed this optimization because this data is also required for `PopulateDefaults`, and it has no effect on getter generation.